docs/src/references/c8y-configuration-management.md

```diff
@@ -64,8 +91,41 @@ $ ls -l /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 -rw-r--r-- 1 tedge tedge 97 Mar 22 14:24 /etc/tedge/operations/c8y/c8y_DownloadConfigFile
 ```
 
+The configuration plugin can also act as configuration proxy for child-devices.
+For that to work for a child device named `$CHILD_DEVICE_ID`,
+two files must be added under `/etc/tedge/operations/c8y/$CHILD_DEVICE_ID`
+in order to declare the associated capabilities to Cumulocity.
+These files are just empty files owned by the `tedge` user.
```

Comment:
Long sentence, and why do you explain here things, that you not explained for the main-device (e.g. about `tedge` user)?

Why not just like that:

```suggestion
Same as for the main-device, those two files must be created for a child-device, but with the `child-id` as subfolder.
```